[DOCS] Simplifies semantic_text tutorial by removing copy_to field (#112864)
elastic/elasticsearch: docs/reference/search/search-your-data/semantic-search-semantic-text.asciidoc

<4> The field to store the text reindexed from a source index in the  <<semantic-text-reindex-data,Reindex the data>> step.
<5> The textual data stored in the `content` field will be copied to `semantic_text` and processed by the {infer} endpoint.
The `semantic_text` field will store the embeddings generated based on the input data.
Copying the `content` field content to the `semantic_text` field is not required for using `semantic_text`.

> However, it is recommended because it makes it possible to perform hybrid search during which you perform lexical search on the text field and semantic search on the semantic_text field.

Reviewer: ```suggestion
However, it is recommended because it makes it possible to perform hybrid search. Hybrid search involves combining lexical search on the `text` field and semantic search on the `semantic_text` field.
```

🚗 :)

        }
      },
      "id": 1713868,

>       "text": "There are a few foods and food groups that will help to fight inflammation and delayed onset muscle soreness (both things that are inevitable after a long, hard workout) when you incorporate them into your postworkout eats, whether immediately after your run or at a meal later in the day. Advertisement. Advertisement."

Reviewer 2: This `text` field should be contained within the `content` object, like so:

```json
"content": {
  "text": <text value>,
  "inference": <inference value>,
}
```

        }
      },
      "id": 3389244,

>       "text": "During Your Workout. There are a few things you can do during your workout to help prevent muscle injury and soreness. According to personal trainer and writer for Iron Magazine, Marc David, doing warm-ups and cool-downs between sets can help keep muscle soreness to a minimum."

Reviewer 2: same here

        }
      },
      "id": 363742,

>       "text": "This is especially important if the soreness is due to a weightlifting routine. For this time period, do not exert more than around 50% of the level of effort (weight, distance and speed) that caused the muscle groups to be sore."

Reviewer 2: same here
